### PR TITLE
create back end structure

### PR DESCRIPTION
**Pull request by yurii-kokhalevych**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVNOD-24)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVNOD-40)

## Code reviewers

- [ ] @yarvanya
- [ ] @yurii-kokhalevych
- [x] @denisbodnar
- [x] @AlbaMontis
- [x] @shyyka
- [ ] @avoytovich
- [ ] @yuliannakorol
- [ ] @IraKim

### Second Level Review

- [x] @lhalam
- [x] @4asof4ik

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo
## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by denisbodnar — DISMISSED** [review]

**Review by shyyka — DISMISSED** [review]

File: config/config.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "development": {
+    "username": "root",
+    "password": null,
+    "database": "database_development",
+    "host": "127.0.0.1",
+    "dialect": "mysql"
```
Comment (AlbaMontis) [inline config/config.json]: Why not "Postgres"?

File: server/models/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var fs        = require('fs');
```
Comment (AlbaMontis) [inline server/models/index.js]: I think we should use let or const.

File: config/config.json (hunk 1 above)
Comment (yarvanya) [reply config/config.json]: I think config.json should be in .gitignore...

**Review by yarvanya — DISMISSED** [review]

File: server/models/index.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+let fs        = require('fs');
```
Comment (4asof4ik) [inline server/models/index.js]: const and one spase before and after "="

File: config/config.json [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (4asof4ik) [inline config/config.json]: should be in .gitignore
and add config_example.json

**Review by 4asof4ik — APPROVED** [review]

**Review by denisbodnar — APPROVED** [review]

**Review by shyyka — APPROVED** [review]

**Review by AlbaMontis — APPROVED** [review]

**Pull request merged**